lib/grammers-client/examples/dialogs.rs

```diff
@@ -42,8 +41,8 @@ async fn async_main() -> Result<()> {
         .init()
         .unwrap();
 
-    let api_id = env!("TG_ID").parse().expect("TG_ID invalid");
-    let api_hash = env!("TG_HASH").to_string();
+    let api_id = std::env::var("TG_ID")?.parse()?;
+    let api_hash = std::env::var("TG_HASH")?;
```

Comment:
`env!` is intentionally used because I think it's good to showcase that the **developer's** API ID and hash must be embedded in the binary (if you ship your application to a customer, they shouldn't have their own API ID and hash in the environment! Official applications also embed the value in the binary). So I would rather we removed this change.